cypress-remote.json

```diff
@@ -0,0 +1,7 @@
+{
+  "projectId": "1ya4vj",
+  "baseUrl": "https://staging.dockstore.org",
```

Comment:
If there are multiple baseUrls to test, then there should really just be the one cypress.json with baseUrl overridden when running the command.